src/main/scala/com/spotify/spark/bigquery/BigQueryClient.scala

```diff
@@ -109,8 +110,29 @@ private[bigquery] class BigQueryClient(conf: Configuration) {
    */
   def load(gcsPath: String, destinationTable: TableReference,
            writeDisposition: WriteDisposition.Value = null,
-           createDisposition: CreateDisposition.Value = null): Unit = {
+           createDisposition: CreateDisposition.Value = null,
+           isPartitionedByDay: Boolean = false): Unit = {
```

Comment:
@nevillelyh what do you think? Add Value in `CreateDisposition` or explicit argument? By adding Value we diverge from the BQ a bit, but makes more sense semantically. I would say be +1 to the new Value.